app/models/scooter.rb

```diff
@@ -1,3 +1,9 @@
 class Scooter < ApplicationRecord
   belongs_to :user
+  has_many :bookings
+  has_one_attached :image
+
+
+  validates :description, presence: true
+  validates :price, presence: true
```

Comment:
here you also called it :price. So in case you decide on :price_per_day you would have to change this line aswell